spec.go

```diff
@@ -18,14 +18,13 @@ import (
 	sdk "github.com/conduitio/conduit-connector-sdk"
 )
 
-// Specification returns the connector's specification.
-func Specification() sdk.Specification {
+// NewSpecification returns the connector's specification.
+func NewSpecification() sdk.Specification {
 	return sdk.Specification{
-		Name:         "notion",
-		Summary:      "A Conduit connector for Notion.",
-		Description:  "A Conduit connector for Notion.",
-		Version:      "v0.1.0",
-		Author:       "Meroxa, Inc.",
-		SourceParams: map[string]sdk.Parameter{},
+		Name:        "notion",
+		Summary:     "A Conduit connector for Notion.",
+		Description: "A Conduit connector for Notion.",
+		Version:     "v0.4.0",
```

Comment:
is this tracking the latest sdk/conduit version? If so, should we change to v0.5.0?

Reply:
@samirketema this is actually the version of the connector itself. A connector version itself doesn't necessarily need to track the SDK version (i.e. they are released completely separately)

Reply:
ah, yes! I didn't realize I forgot to update the version here when I bumped to v0.3.0, so it threw me off to see `v0.1.0` currently. sounds good!